database/metadata/databases/default/tables/public_atd_apd_blueform.yaml

```diff
@@ -8,87 +8,87 @@ insert_permissions:
       columns:
         - address
         - case_id
-        - case_timestamp
+        - date
```

Comment:
I saw you mentioned this table, and I also saw this message when applying metadata. Something is a little off with the column names since `case_timestamp` and `is_deleted` are present in the prod replica.

<img width="1461" alt="Screenshot 2025-04-25 at 6 21 18 PM" src="https://github.com/user-attachments/assets/840bd58c-a2d1-4ea2-a8f1-f461c3ac66ac" />

Reply:
@mateoclarke would you double-check this? looks like your local DB may have been missing [these updates](https://github.com/cityofaustin/vision-zero/pull/1761).